src/site/content/en/blog/wakelock/index.md

```diff
@@ -111,65 +111,52 @@ that your app can continue running.
 
 ### Get a wake lock {: #get-wake-lock }
 
-To request a wake lock, you need to call the `WakeLock.request()` method
-that lives on the `window` object. You pass it the desired wake lock type as
-the first parameter, which *currently* is limited to just `'screen'`.
-You also need a way to abort the wake lock, which works through the
-generic `AbortController` interface. Therefore, you first create a new
-[`AbortController`](https://developer.mozilla.org/en-US/docs/Web/API/AbortController),
-and then pass the controller's
-[`AbortSignal`](https://developer.mozilla.org/en-US/docs/Web/API/AbortSignal)
-as the second parameter to `WakeLock.request()`. Two things can happen next
-that you need to `catch`:
-
-* The wake lock can, after a while, just be regularly aborted, which you
-  detect by checking if the exception's name is `'AbortError'`. In this
-  context, `AbortError` is actually not an error in the common sense, but
-  just the way `AbortController` works.
-* The browser can also refuse the request for various reasons (for example,
-  because the battery charge level is too low). In this case, the exception's
-  message will contain more details.
+To request a wake lock, you need to call the `navigator.wakeLock.request()` method
+that returns a `WakeLockSentinel` object.
+You pass this method the desired wake lock type as a parameter,
+which *currently* is limited to just `'screen'`.
+The browser can refuse the request for various reasons (for example,
+because the battery charge level is too low),
+so it's a good practice to wrap the call in a `try…catch` statement.
+The exception's message will contain more details in case of failure.
+
+You also need a way to release the wake lock, which works through the
```

Comment:
'which works through'

I suggest changing to 'which is achieved by calling'. It sounds less ambiguous to me.

Reply:
Changed, thanks!